src/org/linphone/LinphonePreferences.java

```diff
@@ -693,7 +704,11 @@ public void setAccountEnabled(int n, boolean enabled) {
 	}
 
 	public boolean isAccountEnabled(int n) {
-		return getProxyConfig(n).registerEnabled();
```

Comment:
Besides settings fragment, "isAccountEnabled" is used in other cases, as well, and it will brake some logic.
For example if user is not registered for some reasons it will show disabled in account setting.